### DISCUSSION TITLE
No provider for _OverlayPanel!

### DISCUSSION THREAD

**Discussion by aamatpmo** [primefaces/community#1338] [PrimeNG]
Hi!
I'm trying to use a Menu overlay in my Angular 17 app with standalone components, but I get this error:

ERROR NullInjectorError: R3InjectorError(Standalone[_LayoutComponent])[_OverlayPanel -> _OverlayPanel -> _OverlayPanel]: 
  NullInjectorError: No provider for _OverlayPanel!

I have included the OverlayPanelModule module in the imports of my component.

Any suggestions?

Thanks in advance

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
